.github/workflows/ci.yml

```diff
@@ -101,6 +101,14 @@ jobs:
         flags: unit
         fail_ci_if_error: false
 
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          github-token: ${{ secrets.MERGE_TOKEN }}
```

Comment:
I don't like feeding a PAT here. Why not use the built-in GITHUB_TOKEN?

Reply:
Didn't know what tokens were there (or whether they do the same thing). I'll give this a try.

Reply:
`GITHUB_TOKEN` is auto-provisioned per-job by GHA.